vite.config.js

```diff
@@ -70,7 +70,7 @@ export default defineConfig({
         serveLocalesPlugin(),
         copy({
             targets: [
-                { src: ["locales", "resources", "src/tabs", "src/images"], dest: "src/dist" },
+                { src: ["locales", "resources", "src/tabs", "src/images", "src/resources"], dest: "src/dist" },
```

Comment:
Why is that schema even there? All the other resources are in the root `resources` folder https://github.com/betaflight/betaflight-configurator/tree/master/resources

I'd rather move schema to there, then it would be accessible with simple `/resources/jsonschema/vtxconfig_schema-1.0.json` as url

Reply:
I can't say... but you are right, both folders must be merged. I don't know where have more sense to have the resources, but I will move it to the `/resources` folder if nobody is against.